src/modules/extractors/recipes.extractor.ts

```diff
@@ -211,7 +211,13 @@ export class RecipesExtractor {
     const ticks = isNaN(Number(recipeProperties.ticks))
       ? null
       : Number(recipeProperties.ticks);
-
+    let toolIds: number[] = [];
+    if (recipeProperties.tools) {
+      toolIds = recipeProperties.tools.split(',').map((v) => {
+        const item = this.itemExtractor.getItemByName(v);
+        return item?.id;
+      });
```

Comment:
Should filter out the empty values to ensure the list isn't a bunch of empty entries